src/snowflake/snowpark/_internal/analyzer/select_statement.py

```diff
@@ -348,6 +393,11 @@ def sql_in_subquery(self) -> str:
     def schema_query(self) -> str:
         return self.sql_query
 
+    @property
+    def individual_complexity_stat(self) -> Counter[str]:
+        # SELECT * FROM entity
+        return Counter({ComplexityStat.COLUMN.value: 1})
```

Comment:
`*` is regarded as one column instead of all columns?

Reply:
yes. in terms of compiling complexity `*` is easier than `col1`, ... `col100`. So we need to reflect that